eth_worker/eth_manager/blockchain_sync/blockchain_sync.py

```diff
@@ -133,4 +144,5 @@ def add_transaction_filter(contract_address, contract_type, filter_parameters, f
     if not persistence_module.check_if_synchronization_filter_exists(contract_address, filter_parameters):
         # Set max_block to block_epoch to act as a de-factor zero-point
         config.logg.info(f'No filter found for address {contract_address} with parameters {filter_parameters}. Creating.')
-        return persistence_module.add_transaction_filter(contract_address, contract_type, filter_parameters, filter_type, decimals, block_epoch=block_epoch)
+        persistence_module.add_transaction_filter(contract_address, contract_type, filter_parameters, filter_type, decimals, block_epoch=block_epoch)
+        return True
```

Comment:
All other defs of `add_transaction_filter` return the filter, and uses of it expect the filter. Why is this one now boolean?

Reply:
I think this is from a previous revision? It's returning normally in current [diff](https://github.com/teamsempo/SempoBlockchain/pull/254/files#diff-53f9cba2470dd5f0cd65a798ff6897a1R147)